### DISCUSSION TITLE
Gists + mobile app

### DISCUSSION THREAD

**Discussion by parasense** [community/community#130412] [Mobile]
I would like to read my gist documents in the mobile app, and potentially even edit those gist documents.

Firstly I'm not sure why normal repos are visible within the (Android) mobile app, but gist repos are not?

Secondly, I realize the mobile app probably wasn't designed to be a markdown client, so reading/writing gist documents (presuming markdown) might be out  of scope. That's fine, but that still does not explain why gists are not reachable in the mobile app like normal repos, because someone might like to toggle a gist public/private, or even add/remove files -- activities that would resemble basic repo management features, and less about the content. 

My mobile experience is a large tablet, not a small phone screen, so my app expectation leans towards a full blown kitchen-sink experience, free of the limitations of words like "mobile" which might entails a set of limitations or constraints. Those pretty much don't exist on modern tablets, which are converging towards the "desktop" experience.  

If anybody can enlighten me as to why Gists are not reachible in the moble app, I would very much appreciate your efforts. 
Finally, If you have read this far, I appreciate your time and patience.

In the meanwhile, I'll take my GitHub activities out of the mobile app, and head over to the web-browser experience. 
 
Thanks!

**Comment by stevepopovich** [comment]
Hi @parasense thanks for the feedback. We have not added support for gists yet, but have heard the feedback. I will note this internally and we will write back when this ships. Thanks!

**Reply by kanition** [reply]
Any update?

**Reply by stevepopovich** [reply]
Still in our backlog, unfortunately.

**Reply by adhban0** [reply]
Any update?

**Reply by Akkiraj1234** [reply]
any update? on this i would love to read my gist on phone and write tooo

**Comment by saxbophone** [comment]
I agree 100% with this need —I only downloaded the Github app to access my Gists, and the fact I can't access them on here makes the app pretty useless for me.

**Reply by Akkiraj1234** [reply]
Exactly. I wanted to use Gists as an idea-dump page where I could edit my ideas and research notes/papers for different projects. That way, I could edit them anywhere, even on the metro, and read them whenever I wanted. But I can’t really do that in the GitHub mobile app, so I don’t see much of a point in using the GitHub mobile app, at least for me.
